herbert_core/classes/MPIFramework/@ClusterWrapper/ClusterWrapper.m

```diff
@@ -222,38 +228,83 @@
                 obj.common_env_var_('HERBERT_PARALLEL_EXECUTOR') = obj.matlab_starter_;
             end
 
-            % Split path into cell of paths
-            curr_path = split(path, pathsep);
-            % Filter array for added paths
-            curr_path = curr_path(~startsWith(curr_path, matlabroot));
-            % Join back together
-            curr_path = strjoin(curr_path, pathsep);
+            % Modify MATLABPATH to add path to user code, which user
+            % may want to use in parallel script
+            obj = obj.add_user_path(pc);
+
+            if obj.DEBUG_REMOTE
+                obj.common_env_var_('DO_PARALLEL_MATLAB_LOGGING') = 'true';
+            else
+                obj.common_env_var_('DO_PARALLEL_MATLAB_LOGGING') = 'false';
+            end
+        end
+        %
+        function obj = add_user_path(obj,pc)
+            % Method adds user script path to the path-es available to
+            % parallel workers.
+            %
+            % The scripts should not be within Horace
+            % root folder as there files should already be on path or used
+            % in tests only.
+            %
+            % WARNING!!!
+            % It assumed that user view on the filesystem is the same as
+            % the view from parallel nodes. This probably should be
+            % changed in a future.
```

Comment:
Please ticket this and in the ticket provide an explanation why the change should happen